.env.default

```diff
@@ -1,13 +1,14 @@
 COMPOSE_FILE=./docker/docker-compose.yml:./docker/docker-compose.override.yml
 COMPOSE_PROJECT_NAME=projectname
 PROFILE_NAME=sdd
-MODULES= default_content
+MODULES=default_content
 THEME_NAME=projectname_theme
 SITE_NAME=Site name
 SITE_MAIL=admin@example.com
 PROJECT_INSTALL=
 PHP_VERSION=7
 IMAGE_PHP=skilldlabs/php:72-fpm
+IMAGE_NGINX=skilldlabs/nginx:1.14.1
 IMAGE_FRONT=skilldlabs/frontend:zen
 DB_URL=sqlite:///dev/shm/d8.sqlite
```

Comment:
Why do we have SQLite as default? On projects we use MySQL so wouldn't it be better to have it as default?

Reply:
We should not use mysql by default cos no reason and it slower to start project